core/variant.cpp

```diff
@@ -1632,7 +1633,9 @@ Variant::operator Basis() const {
 		return *_data._basis;
 	else if (type == QUAT)
 		return *reinterpret_cast<const Quat *>(_data._mem);
-	else if (type == TRANSFORM)
+	else if (type == VECTOR3) {
+		return Basis(*reinterpret_cast<const Vector3 *>(_data._mem));
+	} else if (type == TRANSFORM) // unexposed in Variant::can_convert?
```

Comment:
This was already there, but it's not listed as valid in `Variant::can_convert`. Should it be added?